decidim-core/app/packs/src/decidim/sticky_header.js

```diff
@@ -3,12 +3,34 @@
 // They increase the discoverability of the elements in the header.
```

Comment:
Can you add a couple of lines explaining this feature too as we've done with the Sticky headers?

Reply:
Done